### ISSUE TITLE
Extend OCPBalance for full cell

### ISSUE THREAD

**Issue by brosaplanella** [muhammedsogut/Pybamm#33]
Extend the OCPBalance class to fit the half-cell datasets to a full cell measurement.

**Comment by tinosulzer** [comment]
Would this be fitting all 4 stretch / shift factors at the same time? Or something cleverer?

**Comment by brosaplanella** [comment]
Nope, it would literally be that (though cleverer approaches are welcome). It should work, but I haven't checked if there are any identifiability issues.
